GetBuyerBanks/__tests__/handler.test.ts

```diff
@@ -1,10 +1,71 @@
 /* tslint:disable: no-any */
+process.env = {
```

Comment:
you could make the handler independent from the config, so independent from environment variables. So, get config only in the `index.ts` and pass it as a parameter to the handler.

Reply:
TODO in another PR